datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -290,6 +290,19 @@ def service_metadata(self, meta_name, value):
     def check(self, instance):
         raise NotImplementedError
 
+    def set_external_tags(self, external_tags):
+        from datadog_agent import set_external_tags as _set_external_tags
```

Comment:
Why the lazy import?

Reply:
I figured that we wouldn't need the try/catch at the top of the file if we imported directly in this function. If you're using the AgentCheck class(es) it means you're running on A6, so its gauranteed to be there.